docs/tutorial.rst

```diff
@@ -327,7 +336,10 @@ We can find connected entities easily in Kestrel, e.g., child processes created
 of processes, network traffic created by processes, files loaded by processes,
 users who own the processes. To do so, use the ``FIND`` command with a
 previously created Kestrel variable, which stores a list of entities from which
-to find connected entities. Read more in :doc:`language`.
+to find connected entities. Note that not all data sources have relation data,
+and not all STIX-Shifter are mature enough to translate relation data. The data
```

Comment:
Should read "not all STIX-Shifter connector modules are mature enough"

Reply:
Thanks for the catch! Fixed.